omnibus/config/software/datadog-iot-agent.rb

```diff
@@ -112,7 +112,6 @@
     if windows?
       # defer compilation step in a block to allow getting the project's build version, which is populated
       # only once the software that the project takes its version from (i.e. `datadog-agent`) has finished building
-      env['TRACE_AGENT_VERSION'] = project.build_version.gsub(/[^0-9\.]/, '') # used by gorake.rb in the trace-agent, only keep digits and dots
       platform = windows_arch_i386? ? "x86" : "x64"
       command "invoke trace-agent.build --major-version #{major_version_arg} --arch #{platform}", :env => env
```

Comment:
Did you run manual tests to make sure that when building the trace-agent using invoke, we get the correct tags in reported metrics in the Datadog UI and that `trace-agent --version` reports the correct version?

In other words, does the correct version get baked in by build tags?

Reply:
Wonderful thanks!